section.css

```diff
@@ -0,0 +1,58 @@
+
+.Section {
+	margin: 0 auto;
+	text-align: center;
+}
+
+.Section--background{
+	background: #F6F6F6;
+}
+
+.Section__divider {
+	padding-bottom: 90px;
+}
+
+.Section__title {
+	font-size: 30px;
+	padding-top: 70px;
+	padding-bottom: 60px;
+	text-align: center;
+	font-weight: 800;
+	letter-spacing: 1px; 
+}
+
+.Section__text--about {
```

Comment:
Remember that modifiers aren't added at the "element" level in BEM, but rather only on the block level. ".Section__text--about" is being used sort of as an id based on how unique it is by name. Try instead to add a modifier to ".Section" that would apply these stylings to ".Section__text". Try to come up with a name that describes the styling changes rather than the content. So like, ".Section--large-text", or whatever really would describe what makes these stylings different.